pkg/ccl/backupccl/backup_test.go

```diff
@@ -5182,7 +5195,7 @@ func TestPointInTimeRecovery(t *testing.T) {
 		// newbackup.bank could be done here by the operator.
 
 		sqlDB.Exec(t, `DROP TABLE data.bank`)
-		sqlDB.Exec(t, `ALTER TABLE newbackup.bank RENAME TO data.bank`)
+		sqlDB.Exec(t, `CREATE TABLE data.bank AS SELECT * FROM newbackup.bank`)
```

Comment:
Do we need to `DROP TABLE newbackup.bank`?

Reply:
Don't need to since we drop the database right afterwards